openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -75,6 +96,7 @@ def send_credit_notifications(username, course_key):
         'dashboard_link': dashboard_link,
         'credit_course_link': credit_course_link,
         'tracking_pixel': tracking_pixel,
+        'providers': providers_string,
```

Comment:
You should be using ungettext here to account for the fact that languages deal with pluralization differently. See [here](https://github.com/edx/edx-platform/commit/1d1baa50098c1108a4ae7e0c2fc345fbc8dd8a3c#diff-d8e9b05acd3b7dba84709ae01b99053cR874) for an example.